lib/upgrade/etcd/conversionv1v3/converter.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conversionv1v3
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/projectcalico/libcalico-go/lib/apis/v1/unversioned"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+)
+
+type Resource interface {
+	runtime.Object
+	v1.ObjectMetaAccessor
+}
+
+type Converter interface {
+	v1APITov1BackendConverter(unversioned.Resource) (*model.KVPair, error)
```

Comment:
v1Backend -> V1Backend

Reply:
for consistency with the v capitalization you could switch the ordering to apiV1ToBackendV1

but not too fussed

Reply:
You can drop the "Converter" from the interface methods.